packages/webpack/src/buildtime/policyGenerator.js

```diff
@@ -92,7 +96,7 @@ module.exports = {
           // Knowing the actual specifier is not relevant here, they're used as unique identifiers that match between here and dependencies
           specifier: module.userRequest,
           file: module.userRequest,
-          type: 'js',
+          type: isBuiltin(module.userRequest) ? 'builtin' : 'js',
```

Comment:
was this not enough to get rid of `external:../../../node:fs` item in policy?

Reply:
yep it wasn't enough, but looks like the `packages/core/src/generatePolicy.js` update was ^